clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoCommandLine.java

```diff
@@ -381,27 +381,33 @@ private void handleTableCommand() {
     String schema = name.getSchemaName();
 
     Command.setAuthenticationMode(auth, userName);
+    List<String> missingEntities =
+        Stream.of(
+                catalog == null ? CommandEntities.CATALOG : null,
+                schema == null ? CommandEntities.SCHEMA : null)
+            .filter(Objects::nonNull)
+            .collect(Collectors.toList());
 
     // Handle CommandActions.LIST action separately as it doesn't require the `table`
     if (CommandActions.LIST.equals(command)) {
-      List<String> missingEntities =
-          Stream.of(
-                  metalake == null ? CommandEntities.METALAKE : null,
-                  catalog == null ? CommandEntities.CATALOG : null,
-                  schema == null ? CommandEntities.SCHEMA : null)
-              .filter(Objects::nonNull)
-              .collect(Collectors.toList());
       if (!missingEntities.isEmpty()) {
         System.err.println(
             "Missing required argument(s): " + Joiner.on(", ").join(missingEntities));
-        Main.exit(-1);
+        return;
```

Comment:
This should be Main.exit(-1) not return. If you are having an issue with the tests, it is likely an issue with your tests, as I need to fix up a couple of your previous tests.